usage/app/model/SyndicationUsageRequest.scala

```diff
@@ -1,16 +1,20 @@
 package model
 
-import com.gu.mediaservice.model.usage.{SyndicatedUsageStatus, SyndicationUsageMetadata, UsageStatus}
+import com.gu.mediaservice.model.usage.{PendingUsageStatus, SyndicatedUsageStatus, SyndicationUsageMetadata, UsageStatus}
 import org.joda.time.DateTime
 import play.api.libs.json._
 
 case class SyndicationUsageRequest (
   partnerName: String,
+  syndicatedBy: Option[String],
   mediaId: String,
```

Comment:
I think `startPending: Option[Boolean]` seems like a good solution to avoiding specific partners. To clarify: now the match would be against the `startPending` property and in true case it would resolve to a pending usage status and in the false case, a syndicated usage status?